Sources/xcprojectlint-package/ProjectParser.swift

```diff
@@ -92,16 +92,18 @@ public struct BuildConfigurationList: TitledNode {
 public struct BuildFile: Identifiable, CustomDebugStringConvertible {
   public let id: String
   public let key: String
-  public let fileRef: String
+  public let fileRef: String?
```

Comment:
As far as you know, will this always be a `fileRef` _or_ a `productRef`? Does it make sense to use an `enum` to represent that instead of two optionals?

Reply:
As far as I knew yesterday, it was always a fileRef. :-)

I'll update to an enum.